.github/workflows/test.yaml

```diff
@@ -5,14 +5,34 @@ on:
     branches:
     - master
     - develop
+    
   pull_request:
 
+# ensure multiple CI processes are not running analysis on contracts
+concurrency:
```

Comment:
This is really cool, but could you submit this as a separate PR?

Reply:
sure, no problem